watchdog/requesthandler_test.go

```diff
@@ -430,6 +430,40 @@ func TestHealthHandler_SatusMethoNotAllowed_ForWriteableVerbs(t *testing.T) {
 	}
 }
 
+func TestHandler_HasFullPathAndQueryInFunction_WithCgi_Mode(t *testing.T) {
```

Comment:
Since this is not changing behaviour, just documenting it, I think we could raise a separate PR for it and I'll merge right aware since it's not changing anything.